src/main/java/org/mule/extension/db/internal/DbConnector.java

```diff
@@ -46,7 +46,7 @@
 import java.util.List;
 
 /**
- * Connector for connecting to relation Databases through the JDBC API
+ * The Anypoint Database Connector allows you to connect to relational databases through the JDBC API.
```

Comment:
Shouldn't be "to connect to any relational database..."?
(My English is horrible anyways)

Reply:
"any" is not necessary. It's not wrong though. Your English is not horrible.